### ISSUE TITLE
Pagebuilder hiding paragraphs after page load

### ISSUE THREAD

**Issue by websuits** [webiny/webiny-js#2408]
### Version

5.25

### Operating System

OSX

### Browser

Chrome

### What steps will reproduce the bug?

Create a page add multiple paragraphs. At one point changes are not made on first publish, after you edit-publish again will appear. But the it's stops rendering the paragraphs, you can slightly see them on page loading, but after page load some paragraphs will dissapear.

### What is the expected behavior?

Show all the paragraphs.

### What do you see instead?

Nothing, empty <p></p>

### Additional information

A WYSIWYG editor will make it a lot more useful, also adding a class name field for all the element, including paragraphs.

### Possible solution

After discussing with Ben we think react hydration might be the cause.

**Comment by jimjoes** [comment]
+1 - we have a 2xcol grid on a page, with each column containing another grid, with one cell. That cell contains a headline and a paragraph. The paragraph is in the rendered HTML, and you can see it when the page loads, but when the css is applied, it disappears. I think this is because the element is being incorrectly parsed into HTML when it's pre-rendered. The pre-rendered HTML looks likes this (the content is not inside the `<p>` tag):

```
<p></p>
<div><br /></div>
Today, Strabo connects accounts from the following countries: UK, CAN,
USA, IRE, NED, SP, FR, NZ, and AUS. We expect to move next to India and
the Middle East
<p></p>
```


Definitely present in 5.26 as well.

**Comment by rsshilli** [comment]
I have the same problem with version 5.16. It's so weird. Any workaround?

**Comment by jimjoes** [comment]
Not sure about 5.16, but with 5.26 you can use bullets for the copy and remove the bullet itself - for visible, <p> like behaviour. As long as you don't use bullets elsewhere, you can also modify the <li> styles in the theme.

**Comment by data-ux** [comment]
I’m experiencing the same issue on v5.33.2. Paragraphs are visible in page builder, but hidden on the published page.

**Comment by SvenAlHamad** [comment]
@data-ux we have this fixed in the unstable branch at the moment. We expect it to be released in the stable branch towards end of the quarter. 
This is the commit with the fix.
https://github.com/webiny/webiny-js/commit/956625a786fca1e99ee0bf5d08c961c55038dd89
You can use §yarn patch§ to create a patch for that file within your project until we do the official release.

**Comment by data-ux** [comment]
Thanks @SvenAlHamad! I made the same change as the linked commit with `yarn patch` to package @webiny/app-page-builder and the issue is now resolved for me.

**Comment by bchirico** [comment]
I have the exact same problem, but trying `yarn patch @webiny/app-page-builder` I'm not able to find the file mentioned in the commit you mentioned @SvenAlHamad

**Comment by adrians5j** [comment]
Just a note that this issue has been resolved with version 5.34.1.

**Comment by ile** [comment]
Happens here (after the block quote, before the form I was adding): https://dtqdfwu8z3mi4.cloudfront.net/hmm

Version: `Webiny v5.34.7`

@adrians5j
